src/nats.h

```diff
@@ -2602,6 +2604,21 @@ natsOptions_SetExpectedHostname(natsOptions *opts, const char *hostname);
 NATS_EXTERN natsStatus
 natsOptions_SkipServerVerification(natsOptions *opts, bool skip);
 
+typedef struct x509_store_ctx_st X509_STORE_CTX;
+typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
```

Comment:
Remove those 2 lines. This would cause errors otherwise saying that you are redefining them.